pozo/graphs.py

```diff
@@ -297,3 +298,28 @@ def get_theme(self):
                    "name": self._name,
                    }
         return self._get_theme(context=context)
+    
+    def to_lasio_curve_items(pozo_obj,**kwargs):
```

Comment:
hay mucha inconsistencia entre si se usa un array o un valor unico. no entiendo que es la intencion de la interfaz.

Reply:
La función devuelve una lista de CurveItem, puede ser tanto para un array como para un valor único

Reply:
pero que acepta?

Reply:
Se supone que objetos tipo Graph, estoy viendo cómo arreglar eso para que acepte cualquier tipo de objeto de pozo

Reply:
quiero que me digas que va a acceptar la funcion para generar un las de un pozo, explica la interfaz

Reply:
un objeto de pozo (Track, Axes o Graph), la persona puede especificar: mnemonic, depth, value o description. Si no lo especifica el mismo algoritmo soluciona eso. Luego de validar, en el ciclo for se toma el data, unit y mnemonic, en el mismo ciclo se toman esos datos y se crean objetos tipo CurveItem los cuales se agregan a una lista para que al finalizar la función se retorne una lista de CurveItems. Esta función también se podría arreglar para que agregue esa información calculada al objeto LAS que se importó en un principio, se puede añadir esa información al archivo LAS y ya con eso quedaría bien

Reply:
Ok, postergando esto
> Luego de validar, en el ciclo for se toma el data, unit y mnemonic, en el mismo ciclo se toman esos datos y se crean objetos tipo CurveItem los cuales se agregan a una lista para que al finalizar la función se retorne una lista de CurveItems. Esta función también se podría arreglar para que agregue esa información calculada al objeto LAS que se importó en un principio, se puede añadir esa información al archivo LAS y ya con eso quedaría bien

de esto

> mnemonic, depth, value o description

Cuales son los tipos de esos y que es el punto de especificar eso

Reply:
Lo que pasa es que los CurveItem piden mnemonic, unit, value, descr y data, el usuario puede especificar cada uno menos unit porque el algoritmo lo calcula, o puede solo dar la data y el algoritmo se encarga de los mnemonics y unit pero el reso pueden ser ignorados (value y descr). El depth creo que se puede quitar de la función porque en realidad no es necesario por ahora

Reply:
tipos de objetos nativos de pythpn, no son específicos de algún paquete ni nada, como: Strings, arrays, list, esos así

Reply:
pero hombre sea especifico, cuales son los tipos que tu vas a aceptar. no quiero ejemeplo o teorias. me has propuesto unos parametros y quiero saber que vas a aceptar.

Reply:
mnemonic = lista o String
unit = lista o String
Value = lista o String
description = lista o String
data = array